packages/core/src/lib/TwaGenerator.ts

```diff
@@ -256,6 +264,11 @@ export class TwaGenerator {
       await this.generateIcons(twaManifest.maskableIconUrl, targetDirectory, ADAPTIVE_IMAGES);
     }
 
+    // Generate notification images
+    if (twaManifest.monochromeIconUrl) {
+      await this.generateIcons(twaManifest.monochromeIconUrl, targetDirectory, NOTIFICATION_IMAGES);
```

Comment:
I'm wondering if it would be worth parallelizing any of this. I don't think we have an issue with performance at the moment though. @andreban thoughts?

Reply:
I don't think we have perf issues for now. We can parallelize things, but I think this will require change other pieces of code and might be better suited for it's own pull-request.